docs/sdk_users/error_handling.md

```
@@ -0,0 +1,187 @@
+## PrecheckError
+
+`PrecheckError` is raised before a transaction reaches consensus.
+
+**When raised:** The transaction or query fails pre-consensus validation at a node.
+
+**Attributes:**
+
+| Attribute | Type | Description |
+| --- | --- | --- |
+| `status` | `ResponseCode` | Precheck status returned by the node. |
+| `transaction_id` | `TransactionId | None` | Transaction ID associated with the failed request, when available. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)